integration/go.mod

```diff
@@ -10,9 +10,9 @@ require (
 	github.com/go-openapi/strfmt v0.19.5 // indirect
 	github.com/jedib0t/go-pretty v4.3.0+incompatible
 	github.com/onflow/cadence v0.10.2
-	github.com/onflow/flow-go v0.4.1-0.20200715183900-b337e998d486
+	github.com/onflow/flow-go v0.11.1
```

Comment:
```suggestion
	github.com/onflow/flow-go v0.11.1 # replaced by version on-disk
```

Reply:
Maybe even add "via the replace statement at the bottom of the file"